sleap/gui/app.py

```diff
@@ -751,6 +751,20 @@ def new_instance_menu_action():
             "proofreading or reproducibility."
         )
 
+        predictionMenu.addSeparator()
+        add_menu_item(
+            predictionMenu,
+            "training on colab with sample dataset",
+            "Train in Google Colab with sample data..",
```

Comment:
Probably don't need a link to sample data example -- but could maybe point to https://sleap.ai/notebooks/index.html?

Reply:
Better yet: link to the training with default profiles (`Training_and_inference_using_Google_Drive.ipynb`) and training with custom profile (`Training_with_custom_hyperparameters.ipynb`)

Reply:
Good point - done